bitnami/keycloak/templates/configmap-env-vars.yaml

```diff
@@ -20,16 +20,29 @@ data:
   KEYCLOAK_DATABASE_PORT: {{ include "keycloak.databasePort" . }}
   KEYCLOAK_DATABASE_NAME: {{ include "keycloak.databaseName" . | quote }}
   KEYCLOAK_DATABASE_USER: {{ include "keycloak.databaseUser" . | quote }}
-  KEYCLOAK_ENABLE_TLS: {{ ternary "true" "false" .Values.auth.tls.enabled | quote }}
-  {{- if .Values.auth.tls.enabled }}
+  KEYCLOAK_PRODUCTION:  {{ ternary "true" "false" .Values.production | quote }}
+  KEYCLOAK_ENABLE_HTTPS: {{ ternary "true" "false" .Values.tls.enabled | quote }}
+  {{- if .Values.tls.enabled }}
   KEYCLOAK_HTTPS_PORT: {{ .Values.containerPorts.https | quote }}
-  KEYCLOAK_TLS_KEYSTORE_FILE: "/opt/bitnami/keycloak/certs/keycloak.keystore.jks"
-  KEYCLOAK_TLS_TRUSTSTORE_FILE: "/opt/bitnami/keycloak/certs/keycloak.truststore.jks"
+  {{- if or .Values.tls.usePem .Values.tls.autoGenerated }}
+  KEYCLOAK_HTTPS_USE_PEM: "true"
```

Comment:
Is this always `true` or could the user change it and use other kind of certificates ?

Reply:
It is always true if the condition `or .Values.tls.usePem .Values.tls.autoGenerated` is met.

Reply:
I updated the PR to use `ternary`, that way the env variable will be present for both cases, when using JKS and PEM.